docs/src/tutorial/custom-data.rst

```diff
@@ -0,0 +1,162 @@
+Run the workflow using custom data
+==================================
+
+In this tutorial, you will run the workflow using custom focal data in addition to the example reference data. The reference data will serve as background context for the new data.
+
+.. contents:: Table of Contents
+   :local:
+
+Prerequisites
+-------------
+
+1. :doc:`example-data`. These instructions will set up the command line environment used in this tutorial.
+
+Setup
+-----
+
+If you are not already there, change directory to the ``ncov`` directory:
+
+   .. code:: text
+
+      cd ncov
+
+Curate data from GISAID
+-----------------------
+
+.. warning::
+
+   These steps require a GISAID account. `Register <https://www.gisaid.org/registration/register/>`__ if you do not have an account yet. However, registration may take a few days. Follow `alternative data preparation methods <../guides/data-prep.html>`__ if you wish to continue this tutorial in the meantime.
```

Comment:
What if we moved this information to the first step of the "Setup" page (just before installing Nextstrain)? Requesting a GISAID account is such a major part of SARS-CoV-2 analyses, we could prioritize this information and avoid surprising users with this prerequisite in the middle of the tutorial.

Reply:
Moved to prerequisite sections: 563598af

I went for prerequisite section over "Setup" page since the "example data" tutorial can still be completed without a GISAID account.